wear/build.gradle

```diff
@@ -0,0 +1,37 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+android {
+    compileSdkVersion 26
+
```

Comment:
Missing `buildToolsVersion versions.buildTools` here! 
In that way we are unable to launch the project.